### DISCUSSION TITLE
direction property

### DISCUSSION THREAD

**Discussion by vltansky** [tailwindlabs/tailwindcss#6064] [Ideas]
Hi, looks like [`direction`](https://www.w3schools.com/cssref/pr_text_direction.asp ) property is missing. 
Although it can look like not important and rarely used, but in RTL websites it is used widely, as sometimes you need to set some text with direction ltr, for example dates, numbers or ltr text.

**Comment by AliBahaari** [comment]
Wonderful!
It's one of the missed properties which makes Tailwind solitary in some languages.

**Comment by lukewarlow** [comment]
I've made a PR to try and add it: https://github.com/tailwindlabs/tailwindcss/pull/6359

Out of interest is there a reason to use this over using the `dir` attribute directly on the elements you wish to override (I understand there is some difference with inheritance)?

Reason I ask is that it's not possible to detect the value set with the direction property (reason is explained in the PR), which means it wont work with Tailwind's RTL and LTR variants. Would this be a problem for your use case?

**Comment by AliBahaari** [comment]
Sure! In some situations and different screen sizes, you may want to change the direction by breakpoints properties as `sm`, `lg` and etc.
So, using `dir` attribute or `direction` property, may not be a good choice.
